pkg/fuse/fuse.go

```diff
@@ -500,6 +500,7 @@ func Serve(v *vfs.VFS, options string, xattrs, ioctl bool) error {
 		}
 		return fmt.Errorf("fuse: %s", err)
 	}
+	defer fssrv.Unmount()
```

Comment:
This may break upgrade, which should NOT call umount before exit

Reply:
upgrade call os.Exit(1) in installHandler, and wont run this defer function